packages/debug-contracts/src/styles.css

```diff
@@ -0,0 +1,163 @@
+@import "tailwindcss";
+
+@custom-variant dark (&:where(.dark, .dark *, [data-theme=dark], [data-theme=dark] *));
+
+@theme {
+  /* Debug Contracts Light Theme */
+  /* Core semantic colors */
+  --color-dc-primary: #93bbfb;
```

Comment:
Why do we need `-dc` colors here? Why not to use `-sui` colors, which are imported from `components`?